reactivesearch/src/App.js

```diff
@@ -6,11 +6,199 @@ import {
   ReactiveList,
   SingleRange,
   ResultCard,
-  SingleList
+  StateProvider,
+  SingleList, DataSearchProps
 } from "@appbaseio/reactivesearch";
 import AlgoPicker from './custom/AlgoPicker';
 
+import {CollectorModule, Context, InstantSearchQueryCollector, Trail, Query, cookieSessionResolver, ConsoleTransport} from "search-collector";
+
+var OpenLogClient = require('./ts/OpenLogClient.ts').OpenLogClient;
+var OLWriter = require('./ts/OLWriter.ts').OLWriter;
+
+//import OLWriter from 'OLWriter';
+const sessionResolver = () => cookieSessionResolver();
+
+const queryResolver = () => {
+	const params = new URLSearchParams(window.location.search);
+
+	const query = new Query();
+	query.setSearch(params.get("query"));
+
+	return query;
+}
+const debug = true;
+const trail = new Trail(queryResolver, sessionResolver);
+const context = new Context(window, document);
+const writer = new OLWriter('http://127.0.0.1:9200', 'ubl_log', queryResolver, sessionResolver,  debug);
```

Comment:
Let's create a seperate ticket for making this parameterized or otherwise a build artifact.  When we go to deploy this code on a hosted DigitalOcean platform we'll need to have something like http://chorus-os-edition.dev.o19s.com:9200 ;-)

Reply:
Huh?